client/defaults.lua

```diff
@@ -30,7 +30,7 @@ end
 ---@param useOffset boolean?
 ---@return boolean?
 local function canInteractWithDoor(entity, coords, door, useOffset)
-    if not GetIsDoorValid(entity, door) or GetVehicleDoorLockStatus(entity) > 1 or IsVehicleDoorDamaged(entity, door) then return end
+    if not GetIsDoorValid(entity, door) or GetVehicleDoorLockStatus(entity) > 1 or IsVehicleDoorDamaged(entity, door) or IsPedInAnyVehicle(PlayerPedId(), false) then return end
```

Comment:
just `cache.vehicle` will suffice

Reply:
Modified the code so it uses cache.vehicle, thanks :)